src/panels/lovelace/cards/hui-light-card.ts

```diff
@@ -0,0 +1,313 @@
+import { html, LitElement, PropertyValues } from "@polymer/lit-element";
+import { fireEvent } from "../../../common/dom/fire_event.js";
+import { styleMap } from "lit-html/directives/styleMap.js";
+import computeStateName from "../../../common/entity/compute_state_name.js";
+import stateIcon from "../../../common/entity/state_icon.js";
+import { jQuery } from "../../../resources/jquery";
+
+import "../../../components/ha-card.js";
+import "../../../components/ha-icon.js";
+import { roundSliderStyle } from "../../../resources/jquery.roundslider";
+
+import { HomeAssistant, LightEntity } from "../../../types.js";
+import { hassLocalizeLitMixin } from "../../../mixins/lit-localize-mixin";
+import { LovelaceCard, LovelaceConfig } from "../types.js";
+import { longPress } from "../common/directives/long-press-directive";
+
+const lightConfig = {
+  radius: 80,
+  step: 1,
+  circleShape: "pie",
+  startAngle: 315,
+  width: 5,
+  min: 1,
+  max: 100,
+  sliderType: "min-range",
+  lineCap: "round",
+  handleSize: "+12",
+  showTooltip: false,
+};
+
+interface Config extends LovelaceConfig {
+  entity: string;
+  name?: string;
+}
+
+export class HuiLightCard extends hassLocalizeLitMixin(LitElement)
+  implements LovelaceCard {
+  public hass?: HomeAssistant;
+  private _config?: Config;
+  private brightnessTimout?: any;
```

Comment:
let's prefix names of private vars with `_`. I know it's "double", but we're also interacting with JS and they don't know the concept of private.

Reply:
If you don't know the type, you can hover with your mouse on the function (if it's a built-in), otherwise use `unknown`. With `unknown`, you are not allowed to do any operation besides passing it on or comparing it, until you know the type. With `any`, anything goes and we're back to caveman untyped world.

Reply:
Gotcha. I started to do that that but it was a NodeJS Type so I left it off. I added it in because it was accepted with no errors